protos/core.proto

```diff
@@ -144,6 +144,11 @@ message CompileConfig {
   string return_override = 5;
 }
 
+message Schema {
```

Comment:
```suggestion
message SchemaTarget {
```
WDYT?

Reply:
yeah `SchemaTarget` seems better to me, to align with the `Target` naming.